doc/dnf5.conf.5.rst

```diff
@@ -0,0 +1,1366 @@
+..
```

Comment:
I guess we can also add info about processing the configuration options priorities like `MAINCONFIG`, `COMMANDLINE`, `RUNTIME` etc.